docs/src/modules/components/DiamondSponsors.js

```diff
@@ -80,7 +80,7 @@ export default function DiamondSponsors() {
           data-ga-event-action="drawer"
           data-ga-event-label="octopus"
           href="https://octopus.com/?utm_source=materialui&utm_medium=referral"
-          rel="noopener noreferrer sponsored"
+          rel="noopener noreferrer"
```

Comment:
Google seems to recommend having `rel=sponsored` on sponsored links: https://developers.google.com/search/docs/advanced/guidelines/qualify-outbound-links

Reply:
Oh, I missed these, good catch! Looks like they're promoting non-standard html. Can't find the value back in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel.

I'll add disable directives

I feel like the rule could have value to catch typos, but then it shouldn't autofix. Openeing a [ticket](https://github.com/jsx-eslint/eslint-plugin-react/issues/3458).

Reply:
@michaldudak Well spotted!